README.md

```diff
@@ -1,3 +1,10 @@
+![Leanplum](Leanplum.png)
```

Comment:
could you add dimensions to the images please? otherwise the page will jump during loading